src/components/article/viewArticleComponents.js

```diff
@@ -171,18 +173,17 @@ ArticleContent.defaultProps = {
   description: '',
   body: '',
 };
-
 RecentArticles.propTypes = {
-  onChange: PropTypes.func,
-  current: PropTypes.number,
-  total: PropTypes.number,
-  articles: PropTypes.array,
+  onChange: PropTypes.string,
+  current: PropTypes.string,
+  total: PropTypes.string,
+  articles: PropTypes.string,
 };
 RecentArticles.defaultProps = {
-  onChange: () => ({}),
-  current: 0,
-  total: 0,
-  articles: [],
+  onChange: '',
+  current: '',
+  total: '',
+  articles: '',
```

Comment:
You should leave these the way they were

Reply:
Honestly speaking 😕, there is no motive behind me changing these. Let me check in the view single article the way they were

Reply:
What do you think went wrong here @SokoPaulSokool.